config/example_config.yml

```diff
@@ -35,8 +35,14 @@ Data:
         example_type: 'array'
 
         # Optional Boolean. Default: False
+<<<<<<< HEAD
```

Comment:
Merge message accidentally left in.

Reply:
My mistake, not quite sure how that happened (my editor checks for conflicts before doing a merge commit).